go/healthserver/api.go

```diff
@@ -0,0 +1,43 @@
+/*
+	Package healthserver registers HTTP handlers for health and readiness checks
+
+	Package healthserver registers HTTP handlers for the /healthz and
+	/readiness paths. These handlers are always registered (this is done at
+	package init time).
+
+	By default, the /health handler responds with "OK".
+
+	By default, the /readiness handler responds with a HTTP header with
+	"503 Service Unavailable" and followed by "not specified".
+*/
+package healthserver
+
+// SetHealthy will make the /healthz HTTP handler respond with "OK".
+func SetHealthy() {
+	setHealth("")
+}
+
+// SetNotHealthy will make the /healthz HTTP handler respond with a HTTP header
+// with "503 Service Unavailable" and the status string will be returned
+// following the HTTP header.
```

Comment:
Consider mentioning that passing the empty string or "OK" will cause a panic. Consider doing the same for SetNotReady()